.devcontainer/devcontainer.json

```diff
@@ -9,7 +9,10 @@
 	"features": {
 		"ghcr.io/devcontainers/features/desktop-lite:1": {
 			"version": "latest"
+		}, "ghcr.io/devcontainers/features/docker-in-docker:2": { 
+			"version": "latest" 
 		}
 	},
-	"forwardPorts": [6080]
-}
\ No newline at end of file
+	"forwardPorts": [6080],
+    "postCreateCommand": "pip install ."
```

Comment:
Does this mean that this command will execute when Git workspace is up?

Reply:
Yeah after the docker container is spun up and your code is put into the codespace, the final command it'll run will be a pip install.  It should grab all of the required libraries in the pyproject.toml file.

Reply:
Why not put the pip install inside the docker container?

Reply:
I wasn't sure if the code was available inside the docker container while it was building.  Some brief googling said to just run the pip install commands after the devcontainer was built.